air/src/proof/mod.rs

```diff
@@ -212,13 +213,22 @@ fn get_proven_security(
     options: &ProofOptions,
     base_field_bits: u32,
     lde_domain_size: u64,
+    trace_domain_size: u64,
     collision_resistance: u32,
 ) -> u32 {
     let extension_field_bits = (base_field_bits * options.field_extension().degree()) as f64;
+
     let blowup_bits = options.blowup_factor().ilog2() as f64;
     let num_fri_queries = options.num_queries() as f64;
     let lde_size_bits = lde_domain_size.trailing_zeros() as f64;
 
+    // blowup_plus_bits is the number of bits in the blowup factor which is the inverse of
+    // `\rho^+ := trace_domain_size / lde_domain_size`. `\rho^+` is used in order to define a larger
+    // agreement parameter `\alpha^+ := (1 + 1/2m)\sqrt{rho^+} := 1 - \theta^+`. The reason for
```

Comment:
- Should it be `\rho^+ := (trace_domain_size + 2) / lde_domain_size`
- Rather than `\rho^+ := trace_domain_size / lde_domain_size`?

At least that's how it seems to be computed in the code.

Reply:
Yes, thank you. That's the whole point of the change.